examples/rl-ppo-continuous.rs

```diff
@@ -0,0 +1,274 @@
+//! Implements the reinforcement learning algorithm Proximal Policy Optimization (PPO) on random data.
+
+use dfdx::{
+    gradients::Tape,
+    nn::{
+        self,
+        modules::{Linear, ReLU, Tanh},
+        BuildModule, Module,
+    },
+    optim::{Adam, AdamConfig},
+    prelude::{smooth_l1_loss, tensor_collection::TensorCollection, Optimizer, OwnedTape},
+    tensor::{
+        AsArray, Cpu, HasErr, SampleTensor, SplitTape, Tensor0D, Tensor1D, Tensor2D, TensorFrom,
+    },
+    tensor_ops::{Backward, BroadcastTo, MeanTo},
+};
+use std::f32;
+use std::time::Instant;
+
+const STATE_SIZE: usize = 2;
+const INNER_SIZE: usize = 3;
+const ACTION_SIZE: usize = 1;
+
+/// Custom model struct
+#[derive(Clone, Debug)]
+struct Network<const IN: usize, const INNER: usize, const OUT: usize> {
+    l1: (Linear<IN, INNER, f32, Cpu>, ReLU),
+    mu: (Linear<INNER, OUT, f32, Cpu>, Tanh),
+    std: (Linear<INNER, OUT, f32, Cpu>, ReLU), // TODO: should this be SoftPlus?
+    value: Linear<INNER, OUT, f32, Cpu>,
+}
```

Comment:
SplitInto layer was added just for this purpose 😀 
```rust
type Network<const IN: usize, const INNER: usize, const OUT:usize> = (
    (Linear<IN, INNER, f32, Cpu>, ReLU),
    SplitInto<(
        (Linear<INNER, OUT, f32, Cpu>, Tanh),
        (Linear<INNER, OUT, f32, Cpu>, ReLU),
        Linear<INNER, 1, f32, Cpu>
    )>
);
```

Reply:
Oh, thanks :)